src/main/java/seedu/address/logic/Messages.java

```diff
@@ -66,6 +66,15 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code person} for display to the user after executing addMedCon or delMedCon.
+     */
+    public static String formatMedCon(Person person) {
+        final StringBuilder builder = new StringBuilder();
+        builder.append(person.getName());
+        return builder.toString();
+    }
+
```

Comment:
Instead of the success message being, "Added Medical Condition to: [Name]",
maybe we could change it to "Added Medical Condition: [MedCon] to [Name]